Algorithms/Ruby/remove_duplicates.rb

```diff
@@ -0,0 +1,15 @@
+# @param {String} s
+# @return {String}
+def remove_duplicate_letters(s)
+  s = s.chars.sort
```

Comment:
This can be done without sorting if you keep track of all the letters visited in a set